SU2_CFD/src/driver_structure.cpp

```diff
@@ -185,7 +185,7 @@ CDriver::CDriver(char* confFile,
 
   fsi = config_container[ZONE_0]->GetFSI_Simulation();
 
-  if(nZone == SINGLE_ZONE) {
+  if(nZone == SINGLE_ZONE && config_container[0]->GetKind_Solver() == FEM_ELASTICITY) {
     if (rank == MASTER_NODE) cout << "A single zone driver has been instantiated." << endl;
   }
```

Comment:
Yes, thanks for pointing out that output inconsistency. Indeed, the idea is to have no more distinction among single and multi-zone drivers. I fixed the output which now reads "A Fluid zone has been instantiated".